src/data/valid/Database-interleaved.yaml

```diff
@@ -145,7 +41,7 @@ material_processing_set:
     ordered_mobile_phases:
       - type: nmdc:MobilePhaseSegment
         substances_used:
-          - known_as: nmdc:chem-99-000003 # see src/data/valid/Database-chemical_entity_set-1.yaml
+          - known_as: glucose
```

Comment:
@sierra-moxon Looks like we made a fake ID `nmdc:chem-99-000003` .. called it methanol... then used it in a lot of places.
You're not replacing all examples of `nmdc:chem-99-000003` with the exact thing we identified, but rather just throwing in valid enums... 

Is that fair? In that case, I think we can leave these. Especially to show that there's no limitation or validation of what Classes can use what PVs from the enum for this slot.

Reply:
That's right; I might've been too cavalier about the example replacement, but in my mind, I was "just" replacing the id we made in the examples with any of the values in the enumeration.  :)